src/bin/spin.rs

```diff
@@ -42,15 +41,15 @@ async fn main() {
 }
 
 async fn _main() -> anyhow::Result<()> {
-    tracing_subscriber::fmt()
-        .with_writer(std::io::stderr)
-        .with_env_filter(
-            tracing_subscriber::EnvFilter::from_default_env()
-                .add_directive("wasmtime_wasi_http=warn".parse()?)
-                .add_directive("watchexec=off".parse()?),
-        )
-        .with_ansi(std::io::stderr().is_terminal())
-        .init();
+    let mut telemetry_config = match spin_telemetry::config::Config::from_env() {
+        Ok(config) => config,
+        Err(err) => {
+            eprintln!("Failed to load telemetry config: {err}");
+            return Ok(());
+        }
+    };
+    telemetry_config.set_version(VERSION.to_string());
+    let _telemetry_guard = spin_telemetry::init(telemetry_config);
```

Comment:
Maybe I'm just forgetting how we got here but do we still need the telemetry guard? Can we set the globals instead?

Reply:
nm I forgot there is a global shutdown guard